.tekton/update-dependencies.yaml

```diff
@@ -0,0 +1,180 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[push]"
+    pipelinesascode.tekton.dev/on-target-branch: "[refs/heads/*]"
+    pipelinesascode.tekton.dev/task: "[git-clone]"
+    pipelinesascode.tekton.dev/max-keep-runs: "5"
+  name: pipeline-service-update-dependencies
+spec:
+  timeouts:
+    pipeline: "0h20m0s"
+    tasks: "0h5m0s"
+  params:
+    - name: repo_name
+      value: "{{ repo_name }}"
+    - name: repo_owner
+      value: "{{ repo_owner }}"
+    - name: repo_url
+      value: "{{ repo_url }}"
+    - name: source_branch
+      value: "{{ source_branch }}"
+  pipelineSpec:
+    params:
+      - name: repo_name
+      - name: repo_owner
+      - name: repo_url
+      - name: source_branch
+    workspaces:
+      - name: source
+    tasks:
+      - name: git-clone
+        taskRef:
+          name: git-clone
+          kind: ClusterTask
+        workspaces:
+          - name: output
+            workspace: source
+        params:
+          - name: url
+            value: $(params.repo_url)
+          - name: revision
+            value: "origin/$(params.source_branch)"
+          - name: refspec
+            value: "refs/heads/$(params.source_branch)"
+      - name: update-dockerfiles
+        runAfter:
+          - git-clone
+        workspaces:
+          - name: source
+            workspace: source
+        params:
+          - name: repo_name
+            value: $(params.repo_name)
+          - name: repo_owner
+            value: $(params.repo_owner)
+          - name: source_branch
+            value: $(params.source_branch)
+          - name: target_branch
+            value: "robot/$(params.source_branch)/update_dockerfiles"
+        taskSpec:
+          params:
+            - name: repo_name
+            - name: repo_owner
+            - name: source_branch
+            - name: target_branch
+          results:
+            - name: skip-open-pr
+              description: Flag that controls whether or not to open a PR
+              type: string
+          steps:
+            - name: setup-local-repository
+              image: quay.io/redhat-pipeline-service/dependencies-update:main
+              imagePullPolicy: Always
+              script: |
+                #!/usr/bin/env bash
+                set -o errexit
+                set -o pipefail
+                set -x
+                cd "$(workspaces.source.path)"
```

Comment:
You could specify the property workingDir="(workspaces.source.path)" at step, then you can remove this line.

Reply:
Done